src/components/hosts/HostRowDetail.tsx

```diff
@@ -146,6 +149,10 @@ export const HostDetail: React.FC<HostDetailProps> = ({ inventory, host }) => {
         <DetailItem title="CPU architecture" value={inventory.cpu?.architecture || DASH} />
         <DetailItem title="CPU model name" value={inventory.cpu?.modelName || DASH} />
         <DetailItem title="CPU cores and clock speed" value={rowInfo.cpuSpeed} />
+        <DetailItem
```

Comment:
Can we move this to the third column?
This one is under redesign in other open PRs.

Reply:
Done